discovery/eureka/eureka.go

```diff
@@ -162,21 +162,27 @@ func targetsForApp(app *Application) []model.LabelSet {
 			model.AddressLabel:  model.LabelValue(targetAddress),
```

Comment:
targetAddress := net.JoinHostPort(t.HostName, strconv.Itoa(t.Port.Port))

This should also check for nil.

Reply:
Ok. I will check nil.